standard/test/namespace_test.lua

```diff
@@ -0,0 +1,58 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Namespace/testcases
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local ScribuntoUnit = require('Module:ScribuntoUnit')
+
+local Namespace = Lua.import('Module:Namespace', {requireDevIfEnabled = true})
+
+local suite = ScribuntoUnit:new()
+
+function suite:testIdFromName()
+	-- can not really test much here due to the testcase page
+	-- not being in several name spaces at the same time
```

Comment:
I don't really understand this? (is it tests for `isMain()`?)

Reply:
yes